modules/installation-azure-limits.adoc

```diff
@@ -160,16 +160,19 @@ If your applications create more Kubernetes `LoadBalancer` service objects,
 your cluster uses more load balancers.
 endif::upi,ash[]
 ifdef::upi[]
-| 2
+| 3
 ifndef::ash[]
 | 1000 per region
 endif::ash[]
 |Each cluster creates the following
 link:https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview[load balancers]:
 
 [horizontal]
+//`default`:: Public IP address that load balances requests to ports 80 and 443 across worker machines
+//`api`:: Public IP address that load balances requests to ports 6443 and 22623 across control plane machines
 `default`:: Public IP address that load balances requests to ports 80 and 443 across worker machines
-`api`:: Public IP address that load balances requests to ports 6443 and 22623 across control plane machines
+`internal`:: Private IP address that load balances requests to ports 6443 and 22623 across control plane machines
+`external`:: Public IP address that load balances requests to port 6443 across control plane machines
```

Comment:
I think lines 147-179 can be consolidated into this:
```
|Network load balancers
| 3
ifndef::ash[]
| 1000 per region
endif::ash[]
|Each cluster creates the following
link:https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview[load balancers]:

[horizontal]
`default`:: Public IP address that load balances requests to ports 80 and 443 across worker machines
`internal`:: Private IP address that load balances requests to ports 6443 and 22623 across control plane machines
`external`:: Public IP address that load balances requests to port 6443 across control plane machines

If your applications create more Kubernetes `LoadBalancer` service objects,
your cluster uses more load balancers.
```
The changes that required the additional ifdefs are no longer present with these updates, from what I can tell.

Reply:
I thought so. Thank you for verifying.